js/form-utils/valid-room-number.js

```diff
@@ -0,0 +1,11 @@
+export const validRoomNumber = (roomNumber, capacityChildren) => {
```

Comment:
название функций должны подсказывать, что они (функции) делают. в данном случае, функция синхронизирует доступность опции.  можно назвать как-то applyXxx (применить), ensureXxx (убедиться)

```suggestion
export const ensureAvailableCapacilty = (roomNumber, capacityChildren) => {
```